misc/scripts/godot_gdb_pretty_print.py

```diff
@@ -0,0 +1,109 @@
+# Load this file to your GDB session to enable pretty-printing
```

Comment:
Add shebang (and make file executable with `chmod +x`):
 
```suggestion
#!/usr/bin/env python3
#
# Load this file to your GDB session to enable pretty-printing
```

Also, remember to format the file with `black -l120 path/to/file.py` (it's what we use for other scripts in the repository). I also suggest running `isort path/to/file.py`.

Reply:
Fixed.